src/steps/index.test.ts

```diff
@@ -1,98 +1,250 @@
-import { createMockStepExecutionContext } from '@jupiterone/integration-sdk-testing';
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
 
-import { IntegrationConfig } from '../types';
-import { fetchGroups, fetchUsers } from './access';
+import { CalculatedIntegrationConfig } from '../types';
 import { fetchAccountDetails } from './account';
+import { fetchGroups } from './groups';
+import { fetchConfigurations } from './configurations';
+import { fetchEndpoints } from './endpoints';
+import { fetchScanFindings } from './scan-findings';
+import { setupMalwareBytesRecording } from '../../test/helpers/recording';
 
-const DEFAULT_CLIENT_ID = 'dummy-acme-client-id';
-const DEFAULT_CLIENT_SECRET = 'dummy-acme-client-secret';
+const DEFAULT_CLIENT_ID = 'INVALID';
+const DEFAULT_CLIENT_SECRET = 'INVALID';
+const DEFAULT_ACCOUNT_ID = '1ccc6346-b58e-46f5-8110-3b1612d75b18';
 
-const integrationConfig: IntegrationConfig = {
-  clientId: process.env.CLIENT_ID || DEFAULT_CLIENT_ID,
-  clientSecret: process.env.CLIENT_SECRET || DEFAULT_CLIENT_SECRET,
-};
+function createInstanceConfig(
+  overrides?: Partial<CalculatedIntegrationConfig>,
+): CalculatedIntegrationConfig {
+  return {
+    clientId: process.env.CLIENT_ID || DEFAULT_CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET || DEFAULT_CLIENT_SECRET,
+    accountId: process.env.ACCOUNT_ID || DEFAULT_ACCOUNT_ID,
+    ...overrides,
+  } as CalculatedIntegrationConfig;
+}
 
-test('should collect data', async () => {
-  const context = createMockStepExecutionContext<IntegrationConfig>({
-    instanceConfig: integrationConfig,
+jest.setTimeout(20 * 1000);
+
+describe('MalwareBytes', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupMalwareBytesRecording({
+      directory: __dirname,
+      name: 'malwarebytes_recordings',
+      options: {
+        recordFailedRequests: true,
```

Comment:
I _think_ this can be removed... the `recording.har` did not have any non-200 statuses that I could see.